fluent-bit-kinesis.go

```diff
@@ -365,6 +400,9 @@ func unpackRecords(kinesisOutput *kinesis.OutputPlugin, data unsafe.Pointer, len
 
 //export FLBPluginExit
 func FLBPluginExit() int {
+	if metricsServer != nil {
+		metricsServer.Shutdown()
```

Comment:
nit: Do we want the server shutdown to return an error so we can return `output.FLB_ERROR` instead of `FLB_OK`?